.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,43 @@
+name: Feature or enhancement
+description: Submit a proposal for a new CPython feature or enhancement
+labels: ["type-feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
```

The same line break issue as above
```suggestion
      value: >
```